source/release-notes/1.8.txt

```diff
@@ -21,7 +21,7 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-{+c2c-product-name+} Beta Program 
+{+c2c-full-beta-program+}
```

Comment:
I feel like the general documentation updates should go above the beta document release notes

Reply:
nit: Can we also put the many-to-one, a->b->c, and migrationName sections next to each other?

Reply:
I think the other notes section should be above the beta program as well.